src/styles/RippleCatalog.scss

```diff
@@ -30,13 +40,25 @@
   -webkit-user-select: none;
   background-color: #ffffff;
   outline: none;
+  overflow: hidden;
```

Comment:
interesting...I wonder if we should add overflow: hidden to the `mdc-ripple-radius-bounded` mixin? (This is really a comment for MDC Web core code, but I noticed it here)

Reply:
Yeah, see https://github.com/material-components/material-components-web/issues/2203